### ISSUE TITLE
Clojure 1.12

### ISSUE THREAD

**Issue by eneroth** [greglook/cljstyle#109]
It seems that `cljstyle` has some issues with the new 1.12 interop stuff:

```Clojure
clojure.lang.ExceptionInfo: Invalid symbol: Byte/1.
{:type :reader-exception, :ex-kind :reader-error}Error while processing file src/app/api/okhttp.clj
```

**Comment by jarppe** [comment]
Now that every editor has support for this format cljstyle is the only tool preventing the use of this new format.

**Comment by greglook** [comment]
This is next on my list - haven't had a lot of time for my OSS projects between work and kids lately.

**Comment by greglook** [comment]
Fixed in [0.17.642](https://github.com/greglook/cljstyle/releases/tag/0.17.642)
